src/apps/common/submodules/appSelector/appSelector.js

```diff
@@ -137,6 +147,17 @@ define(function(require) {
 					self.appSelectorSetStore('apps', _.keyBy(appList, 'id'));
 					self.appSelectorSetStore('selectedAppIds', selectedAppIds);
 
+					// Filter apps
+					if (!_.isEmpty(showOnlyAppIds)) {
```

Comment:
This check seems too strict, you should be able to specify an empty array of ids (btw, do you know if the common control has an empty state? I know it was not part of the mockup/spec).

Here, we need to check if the `showOnlyAppIds` is not undefined and an array:
```suggestion
					if (
						!_.isUndefined(showOnlyAppIds)
						&& _.isArray(showOnlyAppIds)
					) {
```
And simply use the default value set by `lodash#get` when declaring `showOnlyAppIds` (which is `undefined`).

Reply:
The common control does not have an empty state, but it will render empty if no apps are listed, due to no existing apps for the specified scope or due to the restrictions.

Reply:
Okay, I'll create a ticket and ping Josh about it.